.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
```

Comment:
Is this right? What is the github-actions ecosystem? Is that like updating the actions we use automatically?

What about the java dependencies?

Reply:
So you have to opt-into github-actions self-updates, but they automatically send package updates outside of this? I don't really want to add this config file if its for 1-2 packages in our action files. Normally you can't just update actions from major versions from 1 -> 2, 2 -> 3. They almost always have a major bump because of a breaking change.

I'd much rather spend time looking into it handling automatic gradle updates and configuring for that.

Reply:
Removed.